### PR TITLE
fix CopyableInput overflow issue

### PR DESCRIPTION
**Pull request by osdiab**
It would display text underneath the buttons. gross!

### Before...

copyable-input-before

### After!

copyable-input-after

### CODE REVIEW

**Comment by kofi-clever** [comment]
Nice! That's been bugging me :)
If you're up for some DTEC, TextInput needs the same treatment for when the "Show" button is enabled. Might even be able to fix it there for both

File: docs/components/CopyableInputView.jsx [hunk 1]
```diff
@@ -69,6 +72,15 @@ export default class CopyableInputView extends Component {
           <label className={cssClass.CONFIG}>
             <input
               type="checkbox"
+              checked={this.state.readOnly}
+              onChange={({target}) => this.setState({readOnly: target.checked})}
+            />
+            {" "}
+            Read Only
```
Comment (kofi-clever) [inline docs/components/CopyableInputView.jsx]: Could you add this to TextInputView as well? I forgot about readOnly

**Comment by osdiab** [comment]
@kofi-clever just realized a problem: if there is an error or a required flag on the `CopyableInput`, those aren't going to show in the top right corner anymore, but to the left of the buttons :P If we wish to keep the semantics of CopyableInputs being validatable, I think a more flexible solution needs to involve providing custom buttons to the TextInput object.

**Comment by osdiab** [comment]
I do want to get this resolved quickly, as it will show up every time in the Chromebook setup wizard I'm trying to push out quickly.

**Comment by kofi-clever** [comment]
yeah, agreed. i think that's what i also landed on at a previous attempt at refactoring.
definitely no need to block on those issues, especially if this is pressing. we can revisit later. opened https://clever.atlassian.net/browse/FEE-19 earlier to try to address part of the issue

**Comment by osdiab** [comment]
Cool, I think my plan is to merge it in in this form, not worry about the general TextInput case at the moment or the error/required things getting pushed over, and then fix those later. I think the most sensical scenario is probably to just allow string->callback mappings on TextInput and let this be a special case.

**Comment by jacobsimon** [comment]
lgtm!

**Pull request merged**